cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help

```diff
@@ -39,6 +39,7 @@ COMMANDS
                         Buffer definition
         openapi         Generate the Ballerina sources for a given OpenAPI
                         definition and vice versa
+        asyncapi        Generate the Ballerina sources for a given AsyncAPI definition
```

Comment:
```suggestion
        asyncapi        Generate Ballerina sources for a given AsyncAPI definition
```

Reply:
Thanks a lot for the suggestion. Resolved